emails/views.py

```diff
@@ -282,15 +307,14 @@ def _sns_message(message_json):
             status=400
         )
     common_headers = mail['commonHeaders']
-    if 'to' not in common_headers:
-        logger.error('SNS message without commonHeaders "to".')
-        return HttpResponse(
-            'Received SNS notification without commonHeaders "to".',
-            status=400
-        )
+
+    to_address = _get_relay_recipient_from_message_json(message_json)
+    if to_address is None:
+        incr_if_enabled('no_relay_domain_in_recipient_fields', 1)
+        logger.error('No Relay domain in To, Cc, or Bcc fields.')
```

Comment:
```suggestion
```
Is the logger here necessary or is the metric enough?

Reply:
Removed the logger line.